Carbon.Cryptography/Carbon.Cryptography.psd1

```diff
@@ -17,7 +17,7 @@
     RootModule = 'Carbon.Cryptography.psm1'
 
     # Version number of this module.
-    ModuleVersion = '2.3.0'
+    ModuleVersion = '3.0.0'

```

Comment:
I'm assuming version 2.x line should be not used?

Reply:
Yes. I changed all the command names, an backwards-incompatible change, so I had to rev the version to 3.0.